js/lib/ui.js

```diff
@@ -418,11 +418,23 @@ elgg.ui.initAccessInputs = function () {
 		}
 		var $select = $(this),
 			acl = $select.data('group-acl'),
-			$note = $('.elgg-input-access-membersonly', this.parentNode);
+			$note = $('.elgg-input-access-membersonly', this.parentNode),
+			$commentCount = $select.data('comment-count'),
```

Comment:
I've removed the $ - do we have any guidelines for when to use camelcase or not for such things?